ads/forkmedia.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function forkmedia(global, data) {
+  validateData(data, ['product']);
```

Comment:
did you want to validate `data-html-access-allowed` here?

Reply:
Tried adding `validateData(data, ['htmlAccessAllowed'])` after Line 27 but validation is failing.

Reply:
ok, thanks for checking. Never mind then :)